### PR TITLE
Issue #92: Remove references to Drupal.org.

### PR DESCRIPTION
**Pull request by hellomrcat**
PR for https://github.com/backdrop/backdrop-issues/issues/92

### CODE REVIEW

File: profiles/standard/standard.install [hunk 1]
```diff
@@ -193,7 +193,7 @@ function standard_install() {
 
   // Create an image field named "Image", enabled for the 'article' content type.
   // Many of the following values will be defaulted, they're included here as an illustrative examples.
-  // See http://api.drupal.org/api/function/field_create_field/7
+  // See https://api.backdropcms.org/api/backdrop/core%21modules%21field%21field.crud.inc/function/field_create_field/1
```
Comment (docwilmot) [inline profiles/standard/standard.install]: api.backdropcms.org makes strange URLs (`/core%21modules%21field%21field.crud.inc` etc). Maybe that should be fixed before adding this to docs.

**Comment by klonos** [comment]
You can write that URL as `https://api.backdropcms.org/api/backdrop/core!modules!field!field.crud.inc/function/field_create_field/1`. It'll still work fine. Sometimes some browsers encode URLs in a funny way when you copy/paste them, but they still work fine.

**Comment by docwilmot** [comment]
better, but still should be `See https://api.backdropcms.org/api/function/field_create_instance/1`

**Comment by hellomrcat** [comment]
Fixed, thanks @docwilmot and @klonos :)

**Comment by quicksketch** [comment]
This looks great. Thanks @hellomrcat for the work and both @docwilmot and @klonos for the reviews!

**Pull request merged**